lib/playground.dart

```diff
@@ -297,6 +294,46 @@ class Playground {
     querySelector("#consoletab").setAttribute('selected','');
   }
 
+
+  _handleAutoCompletion(KeyboardEvent e) {
+    // If we're already in completion bail.
+    if (_isCompletionActive) return;
+
+    if (context.focusedEditor == 'dart') {
+      if (e.keyCode == KeyCode.PERIOD) {
+        editor.completionAutoInvoked = true;
+        editor.execCommand("autocomplete");
+        new Timer(const Duration(milliseconds: 500), _handleHelp);
```

Comment:
Can we determine when the codemirror popup is added to the DOM? Is there a codemirror event we can listen for?